rollup.config.js

```diff
@@ -44,7 +44,47 @@ export default [
       })
     ]
   },
-  
+
+  // browser-friendly UMD build for p5
+  {
+    input: 'targets/p5.js',
```

Comment:
Good thinking here. This is something we could probably also do for our other targets like threejs